doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -31,9 +57,9 @@ The FIWARE NGSI (Next Generation Service Interface) API defines
 (whether to separate the two interfaces is currently under discussion).
 * a set of typical **roles** played by NGSI-compliant components
 
-### Context data modelling and exchange
+## Terminology
```

Comment:
I have kept the current section division .fixed in https://github.com/jmcanterafonseca/fiware-orion/commit/01c2427080c7f44b53a89e26ce38de3faf33d990